src/modelconfig.hpp

```diff
@@ -163,6 +163,26 @@ class ModelConfig {
          */
     std::string customLoaderOptionsStr;
 
+    /**
+         * @brief Flag determing if model is of a stateful flag
```

Comment:
I think there should be determining instead of determing. 
"model is of a stateful flag"?